src/index.ts

```diff
@@ -185,13 +185,20 @@ function extractAppLogTime(log: string): number | undefined {
 }
 
 function extractSquidLogTime(log: string): number | undefined {
-  const regex = /^(\d+\.\d{3})/
-  const extractedTime = regexTimeFromLog(regex, log)
-  if (extractedTime === undefined) {
-    return undefined
-  }
+  let extractedTime = regexTimeFromLog(SQUID_ACCESS_LOG_FORMAT_REGEX, log)
+  if (extractedTime !== undefined) {
+    return Number(extractedTime[1])
+  } else { // try to extract a time from a cache log
+    const cacheLogRegex = /(?<year>\d+)\/(?<month>\d+)\/(?<day>\d+) (?<hours>\d+):(?<minutes>\d+):(?<seconds>\d+)/
+    extractedTime = regexTimeFromLog(cacheLogRegex, log)
+    if (extractedTime === undefined) {
+      return undefined
+    }
+    const { year, month, day, hours, minutes, seconds } = extractedTime.groups!
+    const dateTimeString = `${year} ${month} ${day} ${hours}:${minutes}:${seconds}`
```

Comment:
I'd probably put `/` between the year month and day, just because it looks weird without it, I opened a node console to check and generally better not to surprise future devs

Reply:
That's good point, not sure why I've just left spaces, I'll format it as per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format which is the prescribed format for Date.parse.